build.gradle

```
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.0" // compile against 4.0.0, but maintain compat to 3.4.0
+        classpath "com.android.tools.build:gradle:4.1.0-beta04" // compile against 4.0.0, but maintain compat to 3.4.0
```

comment needs adjusting
```suggestion
        classpath "com.android.tools.build:gradle:4.1.0-beta04" // compile against 4.1.0-beta04, but maintain compat to 3.4.0
```